katzenmint/config/config.go

```diff
@@ -31,6 +31,7 @@ const (
 	absoluteMaxDelay            = 6 * 60 * 60 * 1000 // 6 hours.
 )
 
+// TODO: do we need to cap the default value of LambdaP/LambdaL/LambdaD/LambdaM max delay?
```

Comment:
Yes but ideally, this would be dynamically adjusted overtime. As for a default value, we should be able to calculate default parameters for a specific network configuration.